modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationQueueEntryLocalServiceTest.java

```diff
@@ -53,6 +53,25 @@ public class NotificationQueueEntryLocalServiceTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Test
+	public void testAddNotificationQueueEntry() throws Exception {
+		Assert.assertEquals(
+			0,
+			_notificationQueueEntryLocalService.
+				getNotificationQueueEntriesCount());
+
+		NotificationQueueEntry notificationQueueEntry =
+			_addNotificationQueueEntry();
+
+		Assert.assertEquals(
```

Comment:
I'm thinking if it would be interesting to assert some properties, because I think this test would not aggregate value only asserting if it was added :thinking: 
What do you think?

Reply:
Perfect! I agree